roles/rails/tasks/main.yml

```diff
@@ -72,6 +72,12 @@
     regexp: '^{{ env }}:\n  # secret_key_base: ""'
     replace: '{{ env }}:\n  secret_key_base: "{{ secret_key_base.stdout }}"'
 
+- name: Update application timezone configuration in secrets.yml
+  replace:
+    path: "{{ shared_dir }}/config/secrets.yml"
+    regexp: '^{{ env }}:\n  # time_zone: ""'
+    replace: '{{ env }}:\n  time_zone: "{{ application_timezone }}"'
+
```

Comment:
Just noticed that, in the block before this one, we expect `^{{ env }}:\n  # secret_key_base:`, so I guess these lines won't work :thinking:.

Reply:
Hi @javierm, I've changed the location of the secrets entry so it does not cause issues with `secret_key_base`.  This version replaces all occurrences of `# time_zone` in the `secrets.yml` file.